rdata.c

```diff
@@ -798,6 +798,22 @@ rdata_svcparam_alpn_to_string(buffer_type *output, uint16_t val_len,
 	return 1;
 }
 
+static int
+rdata_svcparam_dohpath_to_string(buffer_type *output, uint16_t val_len,
+	uint16_t *data)
+{
+	uint8_t *dp = (void *)data;
+
+	assert(val_len > 0); /* Guaranteed by rdata_svcparam_to_string */
+
+	buffer_write_u8(output, '=');
+	buffer_write_u8(output, '"');
+	buffer_printf(output, "%s", dp);
```

Comment:
Please some type of isprint check on characters, so that binary data results in printable text on the output. Like, by escaping the characters if they are not printable. Like in rdata_text_to_string(), the characters are escaped if needed.